jetty-core/jetty-http/src/main/java/org/eclipse/jetty/http/HttpField.java

```diff
@@ -668,4 +668,47 @@ public long getLongValue()
             return _long;
         }
     }
+
+    public static class MultiHttpField extends HttpField
```

Comment:
Make it `private` as we don't want people start using this class if then we change the implementation.

Reply:
best I can do is package protected.